frontend/packages/fonts/package.json

```diff
@@ -0,0 +1,100 @@
+{
+  "name": "@code-dot-org/fonts",
```

Comment:
This looks super awesome and super helpful and makes our lives so much easier!
A followup question/discussion: I think it would be awesome to make this package a public npm package as well since it so much helpful and basically allows us to setup design system and fonts in a seamless way. What do you think?

Reply:
We can, in the same way we do for component library. I think we can roll that into the discussion about whether or not to publish these publicly.